<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGuiApplicationClass</class>
 <widget class="QMainWindow" name="QtGuiApplicationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>962</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtGuiApplication</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="MyViewer" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>821</width>
      <height>661</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="xCoord">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>10</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="yCoord">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="zCoord">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>70</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>100</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removePointButton">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>130</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>rm</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="removeAllPointsButton">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>130</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>rm -a</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="createCubeChain">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>170</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Create cube chain</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createFilledCube">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>200</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Create filled cube</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createFilledSphere">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>230</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Create filled sphere</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createSphereEps">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>260</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Create sphere eps</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>962</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyViewer</class>
   <extends>QWidget</extends>
   <header>myviewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="QtGuiApplication.qrc"/>
 </resources>
 <connections/>
</ui>
